doc/sphinx/Pacemaker_Explained/collective.rst

```diff
@@ -274,7 +275,7 @@ _____________
    |                   |                 |    single: option; clone-node-max (clone)             |
    |                   |                 |    single: clone-node-max; clone option               |
    |                   |                 |                                                       |
-   |                   |                 | If ``globally-unique`` is **true**, the maximum       |
+   |                   |                 | If the clone is globally unique, this is the maximum  |
```

Comment:
The one technicality might remain: `clone-node-max=0` and `promoted-node-max=0` might apply even in the case of anonymous clones. We don't necessarily have to cover that in the doc, since it would be a pointless configuration (there are simpler ways).